litentry/primitives/src/assertion.rs

```diff
@@ -33,8 +34,8 @@ pub enum Assertion {
 	A2(ParameterString, ParameterString), // (guild_id, user_id)
 	A3(ParameterString, ParameterString), // (guild_id, user_id)
 	A4,
-	A5(ParameterString, ParameterString), // (twitter_account, tweet_id)
-	A6,
+	A5(Identity, ParameterString), // (twitter_account, tweet_id)
+	A6(Identity),
```

Comment:
I think there's some misunderstanding here (either from my or your side).

A5: the first parameter should be the target twitter account. We want to check if the user has any linked twitter account that follows that account. I see in code the target account is hardcoded as `litentry`, which is incorrect. It should come from this parameter. 

linked account eligibility: **any** linked twitter account. 

A6: number of followers of user's linked twitter accounts. No parameter should be needed - I'm not allowed to check someone else's twitter followers. 

linked account eligibility: **all** linked twitter accounts (so the sum).

I suggest you clarify this with @jingleizhang again.

Reply:
A5: The target twitter account should not be hardcoded as `litentry`, should be passed from FE in parameter.

A6: whether we allow one user to link only one twitter account or multiple accounts, I will ask Kayla right now.

Reply:
A6: the answer is yes, we allow one user to link multiple accounts

Reply:
I believe Kayla asked me before 😂 so it's actually decided by us